node-mixin/rules/rules.libsonnet

```diff
@@ -0,0 +1,106 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'node-exporter.rules',
+        rules: [
+          {
+            // This rule gives the number of CPUs per node.
+            record: 'instance:node_num_cpu:sum',
+            expr: |||
+              count by (instance) (
+                sum by (instance, cpu) (
+                  node_cpu{%(nodeExporterSelector)s}
+                )
+              )
+            ||| % $._config,
+          },
+          {
+            // CPU utilisation is % CPU is not idle.
+            record: 'instance:node_cpu_utilisation:avg1m',
```

Comment:
This is not an avg1m, it's an avg_rate1m - but even then it isn't as it's excluding one label. Try to avoid subtraction which takes you away from the source metric names.

Reply:
I guess it's the whole idea to get away from the source metric name, as the goal is to have a "utilisation" metric. I'll change to avg_rate1m. Let's discuss the rest later.